<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_triangle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Треугольник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_square">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Квадрат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rhombus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ромб</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_circle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>круг</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_star5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Звезда 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_star6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Звезда 6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_star8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Звезда 8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rectangle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Прямоугольник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_hexagon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Гексагон</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sizeup">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>SizeUp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sizedown">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>10</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>SizeDown</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveup">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>50</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Moveup</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_movedown">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>80</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>MoveDown</string>
    </property>
   </widget>
   <widget class="QLabel" name="Area">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>610</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Area</string>
    </property>
   </widget>
   <widget class="QLabel" name="Perimeter">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>610</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Perimetr</string>
    </property>
   </widget>
   <widget class="QLabel" name="CenterMass">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>610</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CenterMAss</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveleft">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>50</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>MoveLeft</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveright">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>50</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>MoveRight</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveline">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>190</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>moveline</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drawlineon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>drawon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drawlineoff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>drowOff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rotleft">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>120</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>rotleft</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rotright">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>110</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>rotright</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>161</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>9</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_rotlinel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>92</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>rotlineleft</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rotliner">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>92</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>rotlineright</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sizelinedown">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>92</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>linedown</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sizelineup">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>92</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>lineup</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QDoubleSpinBox" name="TriangleSpinA">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="TriangleSpinB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="TriangleSpinC">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_9"/>
    <widget class="QWidget" name="page_10"/>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4">
     <widget class="QDoubleSpinBox" name="updateSizeR5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QDoubleSpinBox" name="updateSizeR">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QDoubleSpinBox" name="updateSizeR8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QDoubleSpinBox" name="RectSpinL">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="RectSpinW">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QDoubleSpinBox" name="updateHexagon">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1274</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
